packages/graphql-toolbox/src/components/Filename.tsx

```diff
@@ -62,14 +62,14 @@ const Ending = (props: { extension: Extension }) => {
     return <span>{content}</span>;
 };
 
-export const FileName = ({ extension, name, buttons }: Props) => {
+export const FileName = ({ extension, name, buttons, borderRadiusTop = true }: Props) => {
     const theme = useContext(ThemeContext);
 
     return (
         <div
-            className={`w-full h-12 m-0 pt-3 pb-3 pl-4 ${
-                theme.theme === Theme.LIGHT ? "bg-white" : "bg-draculaDark"
-            } rounded-tl-xl rounded-tr-xl flex justify-between items-center`}
+            className={`w-full h-12 m-0 pt-3 pb-3 pl-4 ${theme.theme === Theme.LIGHT ? "bg-white" : "bg-draculaDark"}${
+                borderRadiusTop ? " rounded-t-xl" : ""
+            } flex justify-between items-center`}
```

Comment:
It might be a nice idea to use the [classnames package](https://www.npmjs.com/package/classnames) here, if we did we could change it to something like this:

```suggestion
            className={classNames("w-full h-12 m-0 pt-3 pb-3 pl-4 flex justify-between items-center", theme.theme === Theme.LIGHT ? "bg-white" : "bg-draculaDark", borderRadiusTop && "rounded-t-xl")}
```

Reply:
Oh cool didn't know of that package. Will use!